corrector.rb

```diff
@@ -0,0 +1,6 @@
+class Corrector
+  def correct_name(name)
+    name = name.capitalize
+    name[0, 10] if name.length > 10
+  end
```

Comment:
- [ ] [OPTIONAL] Great job on the corrector class 👏 if you want your code to look a bit cleaner you can remove the if statement and the exclamation point after `capitalize`, then it will look like below:
```
    name.capitalize!
    name[0,10]
```